repo-scripts.install

```diff
@@ -0,0 +1,14 @@
+post_install() {
+  post_upgrade
+  passwd -l repo &>/dev/null
+	#echo -e "Add message"
+}
+
+post_upgrade() {
+  getent group repo >/dev/null || groupadd -g 569 repo
+  getent passwd repo >/dev/null || useradd -c 'Repo Script User' -u 569 -g repo -b '/srv' -m -s /bin/bash repo
```

Comment:
i believe that creation home directory is not required in this case because `/srv` usually provided by filesystem package. Also shell for this user makes no sense in my opinion and should be set to `/usr/bin/nologin`